software/webapp/src/test/java/brooklyn/entity/webapp/WebAppLiveIntegrationTest.groovy

```diff
@@ -81,7 +83,8 @@ public class WebAppLiveIntegrationTest {
         TomcatServer tomcat = new TomcatServerImpl(parent:application, httpPort:DEFAULT_HTTP_PORT, jmxPort:DEFAULT_JMX_PORT)
         JBoss6Server jboss6 = new JBoss6ServerImpl(parent:application, portIncrement:PORT_INCREMENT, jmxPort:DEFAULT_JMX_PORT)
         JBoss7Server jboss7 = new JBoss7ServerImpl(parent:application, httpPort:DEFAULT_HTTP_PORT, jmxPort:DEFAULT_JMX_PORT)
-        return [ [ tomcat ], [ jboss6 ], [ jboss7 ] ]
+		Wildfly8Server wildfly8 = new Wildfly8ServerImpl(parent:application, httpPort:DEFAULT_HTTP_PORT, jmxPort:DEFAULT_JMX_PORT)
```

Comment:
Ah, I see we still do have code that calls entity constructors directly (and that wants to do it for Wildfly8ServerImpl).

We should convert this class to Java, and see about using EntitySpec. But don't want to add yet more to this PR so let's address this another time.